src/Nox.Types/Types/EncryptedText/TypeOptions/EncryptedTextOptions.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System.Security.Cryptography;
+
+namespace Nox.Types;
+
+public class EncryptedTextOptions
```

Comment:
Can we update the Schema for this new TypeOptions?
https://github.com/NoxOrg/Nox.Generator#schemas-update

Reply:
Also we need to call this class "EncryptedTextTypeOptions"

Reply:
Schema updated.
Class name changed.